cyclops-ctrl/internal/cluster/k8sclient/client.go

```diff
@@ -559,6 +561,23 @@ func (k *KubernetesClient) mapConfigMap(group, version, kind, name, namespace st
 	}, nil
 }
 
+func (k *KubernetesClient) mapPersistentVolumeClaims(group, version, kind, name, namespace string) (*dto.PersistentVolumeClaim, error) {
+	persistentvolumeclaim, err := k.clientset.CoreV1().PersistentVolumeClaims(namespace).Get(context.Background(), name, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	return &dto.PersistentVolumeClaim{
+		Group:     	 group,
+		Version:   	 version,
+		Kind:      	 kind,
+		Name:      	 name,
+		Namespace: 	 namespace,
+		AccessModes: persistentvolumeclaim.spec.accessModes,
+		Size: 		 persistentvolumeclaim.spec.resources.requests.storage,
```

Comment:
Can you fix this as well? Also, do we need to checkout for nil pointer dereference if Storage is nil?

Reply:
we need deleted value populated here as false?

Reply:
Also I think storage is a necessary field in PVCs so might never be nil

Reply:
> Also I think storage is a necessary field in PVCs so might never be nil

I was thinking the same thing, but would still like to be cautious since we are dealing with pointers. If its nil, you can default to 0

Reply:
Added